### DISCUSSION TITLE
媒體識讀

### DISCUSSION THREAD

**Discussion by hardtagottaname** [1101-GS4719-C/Python-Community-GS4719-C#1348] [General]
現今社會網路過於發達，一個訊息經過多人在社群媒體上的傳播，擴散力不容小覷。而媒體新聞工作者大部分也為了達到他們的業績，也就是我們所說的流量，不擇手段，不經求證就將小道消息做成一篇篇的媒體報導，而觀看報導者也因為缺乏辨別能力，以訛傳訛，對於被報導者言語攻擊、網路暴力，最終導致慘劇。
因此，我認為，在接收消息的同時，我們應該要懷著懷疑的態度看待每一件事，不能一昧地相信。除此之外，我們也要多多吸收其他人的觀點，才能培養用不同角度觀看同一件事。

**Comment by ziihui** [comment]
非常贊同你的觀點。在這個充滿訊息的時代，我們必須懷著懷疑的態度來看待每一則消息，尤其是在社交媒體上，許多訊息都經過多次轉發和修改，甚至有些是假消息，容易引起誤解和不良影響。

作為接收者，我們可以採取多方面的方法來確認消息的真實性，例如查閱多個可靠的新聞來源、尋找專家意見、搜尋相關資料等。此外，我們也應該保持開放心態，不斷吸收來自不同觀點的訊息，透過不同角度的分析，才能得到更全面的資訊。

**Comment by KevinCheng42** [comment]
在上完大眾傳播與生活通識課後，我也從傳播理論中了解媒體識讀的重要性，除了多方查詢不同立場的新聞以尋求較準確的消息外，我們也可以了解媒體的立場對社會上的影響以及其目的為何，避免掉入他們的陷阱中～

**Comment by llh26** [comment]
光看娛樂新聞就知道假新聞的問題有多嚴重了。現在這個時代，人手一支手機，大家都可以在網路上發表自己的看法和意見。隨著科技發展，也讓照片、影片這類的換頭技術越來越成熟，人們用肉眼根本無法辨別事情的真偽，甚至以訛傳訛讓訊息快速流傳。我覺得人們不能因為是在網路上便為所欲為，無論在何處、透過任何媒介，都應該負起言論責任。

**Comment by brian431** [comment]
除了媒體試讀能力，媒體本身對自己所釋出的報導負責任也是非常重要的。現在實在是有太多不經過仔細求證就輕易發報導的媒體了，多到一個很誇張的程度。然而民眾也不是都有這麽多時間去將每一個新聞的真實性做確認，所以最根本的還是媒體不要亂發新聞。當然最基本的識讀能力還是要有，不過有些資訊是在也不知道是真是假，也只能抱著懷疑態度了。

**Comment by CCello** [comment]
您所提出的觀點非常有道理。在今天的社會中，信息爆炸，很容易受到各種消息的干擾和誤導。因此，我們應該保持警覺和懷疑的態度，不要輕易相信所看到或聽到的信息，特別是那些沒有經過求證的消息。在接收消息時，我們需要進行評估和分析，嘗試將其與其他可靠的資訊進行比較，從而得出一個更為全面和客觀的結論。

**Comment by a3240281370** [comment]
現在真的太多假新聞了！媒體識讀的能力真的很重要，有效的媒體識讀包括查證消息來源和作者的可信度、對比多個媒體來源，確保信息的一致性，隨時注意新聞報導中的情感色彩和立場偏見，也可以辨識常見的假新聞特徵，如拼錯字、荒謬的主張或缺乏可靠來源，最重要的是養成批判思維，思考新聞的合理性和可能的動機。培養良好的媒體識讀能力是保持信息真實性和對抗假新聞的關鍵。

**Comment by Superlan8** [comment]
假新聞近幾年來真的越來越多，尤其選舉又即將到來，我們真的要動腦去思考種種議題，不要被輿論帶著走，我可以提供幾個之前學到的幾個媒體識讀的方法。

1. 多找相關訊息的來源：從不同的媒體來源獲取資訊是最主要的，多元化的觀點和意見有助於獲得更全面的圖像。
2. 想想訊息隱含的目的：了解媒體內容背後可能存在的利益和目的，有些內容可能是廣告或宣傳，有特定的意圖，這種訊息的真實性就要打個大問號。

**Comment by Brownnnn78** [comment]
我真的覺得這問題很重要，每個人都應該要自己學會判斷事情的真假，尤其是長輩經常在賴群到處轉發奇怪的文章，真的是想哭，資訊越來越發達，如果以前沒有公民老師教我媒體識讀這部分的東西我還真的會盲目地相信許多怪資訊。

**Comment by zephyrxin01** [comment]
這個世代資訊量實在是太多了，媒體識讀成為一個超級重要的能力，我統整了一些培養媒體識讀能力的方式，包括：避免局限在單一來源或類型的媒體、學習以批評的角度去接觸資訊而不是全盤接受、確定資訊來源是否可靠，甚至能以交叉比對的方式去確認。
我覺得以上這些方法對我來說很有用，提供給大家參考。

**Comment by sumairu0727** [comment]
我認為媒體識讀會出現問題是因為人們在事情還沒有結果之前就喜歡在旁邊當吃瓜群眾，而媒體抓住這個現象一直去評論那些根本還沒有結論的事情，又加上標題殺，促成媒體識讀低落。
我認為增加媒體識讀的能力應該要從「不要隨意評斷根本還沒有結果的事」做起，不然整件事很容易被媒體拿出來帶風向，這樣對當事人跟社會都不會有好得影響。

**Comment by pudding0803** [comment]
在 AI 的創作越來越普及的同時，人們在判斷真假時也變得越來越困難，因此現在可能變成要不是有非常可信的證據（科學證明、實驗結果、親眼為證等），一切都不要相信太滿為妙。之前有聽說過有學校開課教導學生媒體識讀相關的能力，我認為學校課綱應該要與時俱進，尤其在這大幅改變的時代，或許一些相關的新世代知識、問題，還是要盡早交給學生較好。

**Comment by youtinglin** [comment]
蠻認同作者的觀點，其實不管是在社群媒體上的消息要抱持懷疑的態度，在生活中的所見所聞也要清楚地審視，因現代人很容易跟風，事實上多數人做的事或說的事不一定就是正確的，不能盲目地相信，要具備自我判斷的能力，也不要因不清楚事情的來龍去脈就隨意的批評，雖然每個人都有言論自由，但發表言論時還是要注意禮貌和分寸避免傷害到他人。

**Comment by Elaine2899** [comment]
非常認同這樣的觀點，站在不同的角度才能看到不一樣的視野，而不同的視野才能提供我們更加正確的解讀，就像近年因為網路發達，導致各種事件的發展容易由單方釋出資訊，大家就只會看到一方面的解讀，然後在網路上霸凌另一方，但等到另一方釋出他掌握的資料時，就會發現局勢顛覆了，原來先前發布消息的才是加害者。像這樣的事件發生時，媒體識讀就非常重要，才不會讓我們對被害人造成二次傷害。

**Comment by MaxChi870** [comment]
現今人們對於社會的資訊多來自媒體的傳播，但媒體資訊的來源正確與否，以及其的虛實需由接收者去自行吸收判斷，因此培養正確的媒體識讀能力是相當重要的。

**Comment by Alex11392** [comment]
確實，現今媒體的環境充斥主觀或不真實的訊息，十分仰賴訊息接收者的判讀能力

**Comment by ThanatosJun** [comment]
我認為媒體識讀的主要概念就是如何從眾多破碎的訊息間找出正確的碎片，並懂得自主尋找更多的訊息去做對比，找出相似與差異，找出矛盾與相契，這是不是很像偵探辦案呢？
